src/helper.php

```diff
@@ -33,6 +36,30 @@ function tinker(...$args)
             ->combine($args)
             ->all();
 
-        exec('open "tinkerwell://?cwd='.base64_encode(base_path()).'&scope='.base64_encode(serialize($namedParameters)).'"');
+        //exec('open "tinkerwell://?cwd='.base64_encode(base_path()).'&scope='.base64_encode(serialize($namedParameters)).'"');
+
+        // For MacOS
+        $command = 'open "tinkerwell://?cwd='.base64_encode(base_path()).'&scope='.base64_encode(serialize($namedParameters)).'"';
+
+        if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+            $logPath = base_path() . '\\storage\\logs\\tinkerwell-' . date("Y-m-d") . '.log';
+
+            // For Windows || ATTENTION ... I added a ^ before the & to escape it.
```

Comment:
Nice comment! This is an annoying quirk of Windows that I only discovered the other day. 👍